packages/ckeditor5-engine/src/view/observer/selectionobserver.js

```diff
@@ -26,7 +26,7 @@ import { debounce } from 'lodash-es';
  * @extends module:engine/view/observer/observer~Observer
  */
 export default class SelectionObserver extends Observer {
-	constructor( view ) {
+	constructor( view, sourceElementRoot = document ) { // eslint-disable-line no-undef
```

Comment:
why is this lint disable needed here?

Reply:
it was because es-lint was complaining that `document` was undefined. I ended up seeing `/* global document */` at the top of other files, which I added so es-lint no longer complains.